.github/workflows/forward-merge.yml

```diff
@@ -130,7 +130,6 @@ jobs:
           appCode: dlpro96xybh
           storybookBuildDir: docs
           exitOnceUploaded: false
-          exitZeroOnChanges: true
```

Comment:
This told Chromatic to not fail CI if any visual changes were detected. This is not intentional. This causes unreviewed changes in the ancestor builds of pull requests which is confusing inside a pull request.